src/rules/memberOrderingRule.ts

```diff
@@ -487,3 +555,34 @@ function nameString(name: ts.PropertyName): string {
             return "";
     }
 }
+/**
+ * Returns the last element of an array. (Or undefined).
+ */
+function arrayLast<T>(array: ArrayLike<T>): T {
+    return array[array.length - 1];
+}
+
+/**
+ * Array.prototype.findIndex, but the last index.
+ */
+function arrayFindLastIndex<T>(
```

Comment:
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/lastIndexOf ?

Reply:
Yeah, that doesn't support a custom predicate.